Lib/__future__.py

```diff
@@ -68,14 +68,14 @@
 # this module.
 CO_NESTED            = 0x0010   # nested_scopes
 CO_GENERATOR_ALLOWED = 0        # generators (obsolete, was 0x1000)
-CO_FUTURE_DIVISION   = 0x2000   # division
-CO_FUTURE_ABSOLUTE_IMPORT = 0x4000 # perform absolute imports by default
-CO_FUTURE_WITH_STATEMENT  = 0x8000   # with statement
-CO_FUTURE_PRINT_FUNCTION  = 0x10000   # print function
-CO_FUTURE_UNICODE_LITERALS = 0x20000 # unicode string literals
-CO_FUTURE_BARRY_AS_BDFL = 0x40000
-CO_FUTURE_GENERATOR_STOP  = 0x80000 # StopIteration becomes RuntimeError in generators
-CO_FUTURE_ANNOTATIONS     = 0x100000  # annotations become strings at runtime
+CO_FUTURE_DIVISION   = 0x20000   # division
```

Comment:
Although it should not directly matter, I am not comfortable changing the values of all the constants

Reply:
Well they are currently clashing with each other, so in anyway we need to change one side's value. I'm changing these because now we have plenty of space to add new compiler flags and we dont need to change anything everytime a new compiler flag added.

Reply:
The real problem is that we are mixing ast flags and compiler flags. This solution is just a workaround for the problem. Sadly, to preserve backwards compatibility a "real" fix may be difficult. The problem with this solution is also that they can clash again in the future if (unlikely) enough flags are added anyway. It may be ok, but I would like another core dev review to see if they are fine with this